concordium-node/Cargo.toml

```
@@ -18,7 +18,7 @@ instrumentation = ["serde_derive", "gotham", "mime", "gotham_derive", "prometheu
 network_dump = []
 static = [ ]
 profiling = [ "static" ]
-collector = [ "reqwest/default-tls", "serde/derive", "rmp-serde", "gotham", "mime", "gotham_derive", "hyper", "futures" ]
+collector = [ "reqwest/default-tls", "serde/derive", "rmp-serde", "mime", "collector-backend" ]
```

It seems that the collector is not using mime
```suggestion
collector = [ "reqwest/default-tls", "serde/derive", "rmp-serde", "collector-backend" ]
```